### PR TITLE
[DOCS]7.15.2 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #1180.

Preview [here](https://security-docs_1187.docs-preview.app.elstc.co/guide/en/security/7.15/release-notes.html).

### CODE REVIEW

File: docs/release-notes/7.15.asciidoc [hunk 1]
```diff
@@ -3,6 +3,19 @@
...
 == 7.15
 
+[discrete]
+[[release-notes-7.15.2]]
+=== 7.15.2
+
+[discrete]
+[[bug-fixes-7.15.2]]
+==== Bug fixes and enhancements
+* The Analyze event option has been moved from the overflow menu to the Actions column. It now only displays for events that can be analyzed ({pull}115478[#115478]).
```
Comment (jmikell821) [inline docs/release-notes/7.15.asciidoc:13]:
```suggestion
* The Analyze event option has been moved from the overflow menu to the Actions column. It now only displays events that can be opened in the visual event analyzer ({pull}115478[#115478]).
```

**Review by jmikell821 — COMMENTED** [review]
One small comment, otherwise LGTM.

File: docs/release-notes/7.15.asciidoc [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
 == 7.15
 
+[discrete]
+[[release-notes-7.15.2]]
+=== 7.15.2
+
+[discrete]
+[[bug-fixes-7.15.2]]
+==== Bug fixes and enhancements
+* The Analyze event option has been moved from the overflow menu to the Actions column. It now only displays events that can be opened in the visual event analyzer ({pull}115478[#115478]).
```
Comment (benironside) [inline docs/release-notes/7.15.asciidoc:13]: I am curious, why capitalize "Actions column" but not "overflow menu"?

**Review by benironside — APPROVED** [review]
One quick question in the comments but LGTM

**Comment by nastasha-solomon** [comment]
mergifyio backport 7.16

**Comment by nastasha-solomon** [comment]
@mergify backport 7.16

**Pull request merged**